frontend/main.js

```diff
@@ -321,34 +395,54 @@ async function startApp () {
       }
       sbp('okTurtles.events/off', CONTRACT_IS_SYNCING, initialSyncFn)
       sbp('okTurtles.events/on', CONTRACT_IS_SYNCING, syncFn.bind(this))
-      sbp('okTurtles.events/on', LOGIN, async () => {
+      sbp('okTurtles.events/on', LOGOUT, () => {
+        // TODO: This is to be done by the SW
+        Promise.all([
+          sbp('chelonia/reset'),
+          sbp('gi.db/settings/delete', 'CHELONIA_STATE')
+        ]).catch(e => {
+          console.error('Logout event: error deleting Chelonia state')
+        })
+      })
+      sbp('okTurtles.events/on', LOGIN_COMPLETE, (a) => {
+        const state = sbp('state/vuex/state')
+        if (!state.loggedIn) return
```

Comment:
Should probably add a `console.warn` message here because this is quite unexpected